server/server.js

```diff
@@ -41,19 +41,28 @@ async function handleValidRequest(button, numPresses, batteryLevel) {
     let client = await db.beginTransaction()
 
     let session = await db.getUnrespondedSessionWithButtonId(button.button_id, client)
-        
-    if (session === null) {
-        session = await db.createSession(button.installation_id, button.button_id, button.unit, button.phone_number, numPresses, client)
-    }
-    else {
-        session.incrementButtonPresses(numPresses)
-        await db.saveSession(session, client)
-    }
 
     if ((batteryLevel !== undefined) &&
             (batteryLevel >= 0) &&
             (batteryLevel <= 100)) {
-        await db.saveButtonBatteryLevel(button.button_serial_number, batteryLevel, client)
+        if (session === null) {
+            session = await db.createSession(button.installation_id, button.button_id, button.unit, button.phone_number, numPresses, batteryLevel, client)
+        }
+        else {
+            session.incrementButtonPresses(numPresses)
+            session.updateBatteryLevel(batteryLevel)
+            await db.saveSession(session, client)
```

Comment:
It looks like we're now updating the battery level in *only* the session. In this case, I think we should remove the column from the `registry` table. Otherwise it'll get out of date and it will be confusing.

Either that, or both tables should be kept up-to-date.

Reply:
added a drop column on registry to the migration script

Reply:
Could you please make sure that this runs with the new change on `chatbot-dev`? I think just removing 10 from the `migrations` table and re-running `setup_server.sh` should do it.